api/app/src/external/commonwell/document/document-query-sandbox.ts

```diff
@@ -64,5 +71,21 @@ export async function sandboxGetDocRefsAndUpsert({
     }
   }
 
+  await updateDocQuery({
+    patient: { id: patient.id, cxId: patient.cxId },
+    downloadProgress: {
+      total: entries.length,
+      status: "completed",
+    },
+    ...(fhirConvertCount > 0
+      ? {
+          convertProgress: {
+            status: "processing",
+            total: fhirConvertCount,
+          },
+        }
+      : undefined),
+  });
```

Comment:
does this need to be fixed - or is this not an issue for sandbox since download will always be done before conversions?

Reply:
So i updated to count the total xml's to convert - similar to prod. Let me know if that works